services/tier-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/model/Tier.java

```diff
@@ -12,11 +28,13 @@ public final class Tier {
     private final String id;
```

Comment:
We should add `created` and `modified` like we have with the other domain model objects in SaaS Boost to help with audit and debug. See the Tenant or User service for examples.